src/pages/signing_official_console/DAACell.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Checkbox } from '@mui/material';
+import { DAA } from '../../libs/ajax/DAA';
+import { Notifications } from '../../libs/utils';
+
+export default function DAACell(rowDac, researcher, institutionId, daas, refreshResearchers, setResearchers) {
```

Comment:
Would it be more idiomatic to pass all of these arguments as `props` and then destructure them in the assignments below?